tool/dart_dev/config.dart

```diff
@@ -2,14 +2,7 @@ import 'package:dart_dev/dart_dev.dart';
 import 'package:glob/glob.dart';
 
 final config = {
-  'analyze': AnalyzeTool()
```

Comment:
Removed these included globs since dart_dev will figure out the correct defaults automatically. Also, analysis configuration should be done in the analysis_options.yaml